src/compressor/brotli/CMakeLists.txt

```diff
@@ -6,14 +6,14 @@ set(brotli_sources
 )
 include(ExternalProject)
 ExternalProject_Add(brotli_ext
-  DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
+  DOWNLOAD_DIR ${CMAKE_SOURCE_DIR}/src/
```

Comment:
I think putting the external project source code in the source folder makes it clear that it is part of the application code. I am not sure if this is appropriate. Do you have any suggestions?

Reply:
@bi-shun my suggestion is to keep as it is.

ideally, `${CMAKE_SOURCE_DIR}` contains source and only source which is managed by git. the assets downloaded by cmake or other package management system should go to the build directory. and should be identified easily. please note, when build/testing ceph, we need to download thousands of python modules and JavaScript packages, they are all downloaded into the build directory. i don't really want to have a cluttered source directory. just because they are source code.

also i don't think this change is in the scope of your change to fix the build of Brotli compressor, it belongs to its own commit.